awscli/customizations/ec2instanceconnect/websocket.py

```diff
@@ -155,7 +155,7 @@ def connect(self, url, user_agent=None):
         proxy_options = None
         proxy_url = environment.get("HTTP_PROXY") or environment.get("HTTPS_PROXY")
         no_proxy = environment.get("NO_PROXY")
```

Comment:
Let's make sure to add a changelog entry for this fix. From the root of the repository, you should be able to run the `new-change` script to generate one (feel free to use or repurpose this):
```
~/GitHub/aws-cli$ ./scripts/new-change -t bugfix -c '``ec2-instance-connect ssh``' -d 'Fix runtime error when HTTP_PROXY environment variable is set and NO_PROXY is not set for eice connection type #8023'
```
This will add a new json file for this change at `.changes/next-release` that we can than add as part of the PR.